docs/protocols.md

```diff
@@ -27,77 +27,6 @@ If stronger algorithms are needed (for example, AES with 256-bit keys), the [JCE
 
 Protocol uses non-blocking I/O wherever possible which removes the performance bottleneck of the <code>JNLP3-connect</code> protocol.
 
-## Deprecated protocols
```

Comment:
You forgot to update `docs/remoting-3-compatibility.md`.

Reply:
See #365